experimental/learn_descriptors/symphony_lake_parser.hh

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string.h>
+
+#include "symphony_lake_dataset/SurveyVector.h"
+
+namespace robot::experimental::learn_descriptors::symphony_lake_parser {
```

Comment:
In general, the namespace should match the file path to make things easier to find. Remove `symphony_lake_parser` from the namespace.